### DISCUSSION TITLE
Serving Dependency Frontend on non root PATH

### DISCUSSION THREAD

**Discussion by Shojin-Spear** [DependencyTrack/dependency-track#1514] [General]
HI all,
first of all thanks for the amazing project. 

My question is not really an issue but one configuration.

I have Dependency Track up and running, however I need to deploy  the server behind  NGINX RP on a location, like:

https://server/dtrack

I setu th URL on the BASE URL but this not solve the need as the request always return https://server 

I try to rewrite on the NGINX but can´t make working.

Can you please put me on the correct path.

Thanks

**Comment by stevespringett** [comment]
There's a lot of people running DT behind NGINX, with and without OIDC. Most of these folks are in the Slack channel.

https://dependencytrack.org/slack/invite
https://dependencytrack.org/slack

**Reply by rkschwrz** [reply]
@Shojin-Spear Do you find a solution? I facing the same behavior, the BASE URL config is not working.
